modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -0,0 +1,619 @@
+<?php
+/**
+ * Issue tracker
+ *
+ * Handles issue edits and returns data in response to a front end call.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Media
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+/**
+ * Issue tracker
+ *
+ * Handles issue edits and returns data in response to a front end call.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Media
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once "Email.class.inc";
+
+//TODO: or split it into two files... :P
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action == "getData") {
```

Comment:
This "action" field should be required. You can just check of the $_SERVER['REQUEST_METHOD'] to see if it was a GET request or a POST request. GET should never update, and POST always should.